Tasks/Kasinskiy/Task1/task1.py

```diff
@@ -0,0 +1,9 @@
+from decimal import Decimal as dD
+
+initial_deposit = 20000
+interest_rate = 15
+deposit_term = 60
+
+final_income = dD(dD(initial_deposit)*(1+dD(interest_rate)/(100*12))**dD(deposit_term))
+print(final_income.quantize(dD("1.0000")), 'BYN')
```

Comment:
afaik our currency has "kopeiki" fractions as two digits after decimal comma, so it makes sense to round to a second digit and not forth